packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -252,25 +259,30 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
     })
 
     const nonce = await this.signer.getTransactionCount()
```

Comment:
Unrelated to this PR, but why are we even doing this if we are using `populateTransaction`; shouldn't that automatically be handling the nonce selection for us?

Reply:
Yes it definitely should. Just removed it.

I left it in to avoid changing lines of code that I didn't have to, but I think removing the manual nonce setting is a good improvement.